src/ext/tags.js

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2016 Resonance Labs, Inc
```

Comment:
I have no strong opinion on this, but the convention i've seen in many projects is to put this at the top of the file.